mne_bids/tests/test_report.py

```diff
@@ -59,3 +60,40 @@ def test_report(tmp_path):
 analysis (2.0 +/- 0.0 were removed from analysis)."""  # noqa
 
     assert report == expected_report
+
+
+@pytest.mark.filterwarnings(warning_str['channel_unit_changed'])
+def test_report_no_participant_information(tmp_path):
+    """
+    Test that report generated works as intended even when all participant
+    information such as age, gender, handedness is removed from the tsv file.
+    """
+    bids_root = str(tmp_path)
+    raw = mne.io.read_raw_fif(raw_fname, verbose=False)
+    raw.info['line_freq'] = 60
+    bids_path.update(root=bids_root)
```

Comment:
Avoid altering the original
```suggestion
    bids_path = bids_path.copy().update(root=bids_root)
```